mlir/test/Dialect/DLTI/invalid.mlir

```diff
@@ -90,3 +90,132 @@ module attributes { dlti.dl_spec = #dlti.dl_spec<#dlti.dl_entry<"unknown.unknown
   // expected-note@above {{enclosing op with data layout}}
   "test.op_with_data_layout"() { dlti.dl_spec = #dlti.dl_spec<#dlti.dl_entry<"unknown.unknown", 32>>} : () -> ()
 }
+
+// -----
+
+// expected-error@below {{'dlti.target_system_desc_spec' is expected to be a #dlti.target_system_desc_spec attribute}}
```

Comment:
```suggestion
// expected-error@below {{'dlti.target_system_desc_spec' is expected to be a #dlti.target_device_desc_spec attribute}}
```

Or did I misunderstand this? Though the next test suggests that I got it right :)

Reply:
No, this is just testing the error message for the `42` not being an actual attribute.